spec/features/trainees/degrees/editing_degree_spec.rb

```diff
@@ -93,9 +95,13 @@ def trainee
 
   def uk_trainee
     @uk_trainee ||= create(:degree, :uk_degree_type).trainee
+    @uk_trainee.provider = current_user.provider
+    @uk_trainee.save!
   end
 
   def non_uk_trainee
     @non_uk_trainee ||= create(:degree, :non_uk_degree_type).trainee
+    @non_uk_trainee.provider = current_user.provider
```

Comment:
Same for here re: passing provider in the create

Reply:
Sorted!